stock_quant_manual_assign/models/stock_picking_type.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+class StockPickingType(models.Model):
```

Comment:
```suggestion

class StockPickingType(models.Model):
```
Same as what I commented on OCA side.

Reply:
@TimLai125 Thanks for catching this. Fixed it.